spec/rubocop/cop/style/extra_spacing_spec.rb

```diff
@@ -49,6 +49,28 @@
       expect(cop.offenses).to be_empty
     end
 
+    it 'does not skip single line hash literals with symbol keys' do
```

Comment:
Corrected this here and in the Changelog:
https://github.com/bbatsov/rubocop/pull/3565/commits/e33a26e9b69facd11460688bbb6e58722e55a859